cardano-cli/src/Cardano/CLI/Shelley/Run/Key.hs

```diff
@@ -425,6 +430,34 @@ convertITNSigningKey privKey = do
     Just signKey -> Right $ StakeSigningKey signKey
     Nothing -> Left $ SigningKeyDeserializationError keyBS
 
+convertITNExtendedPublicKey :: Text -> Either ConversionError (VerificationKey StakeExtendedKey)
```

Comment:
The other ITN public keys are successfully converted with `convertITNVerificationKey`.